cs50/2017/fall/challenges/speller/check50/speller.c

```diff
@@ -182,6 +182,7 @@ int main(int argc, char *argv[])
         printf("%.5f\n", time_check);
         printf("%.5f\n", time_size);
         printf("%.5f\n", time_unload);
+        printf("CANARY\n");
```

Comment:
This line will need to be after the `WORDS IN TEXT` line, since currently it's inside of a conditional block that only executes during the benchmarking phase of the checks.